pkgs/conf-pkg/changelog/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2022
+
+* [January](2022/JANUARY.rst)
+
```

Comment:
Only the one changelog is required. I believe this file will be populated by us when we release a new package in a week's time
```suggestion
```